robusta_krr/main.py

```
@@ -310,10 +328,14 @@ def run_strategy(
                         strategy=_strategy_name,
                         other_args=strategy_args,
-                    )
+                        publish_scan_url=publish_scan_url,
+                        start_time=start_time,
+                        scan_id=scan_id,
+                        )
                     Config.set_config(config)
-                except ValidationError:
+                except ValidationError as e:
                     logger.exception("Error occured while parsing arguments")
+                    publish_input_error( publish_scan_url, start_time, scan_id, str(e))
```

Flake8 (7.2.0)

[error] whitespace after '('

(E201)